src/routes/api-routes.js

```diff
@@ -9,7 +10,63 @@ const getXibit = (req, res) => {
   res.send('Get 1 xibit');
 };
 
+const getAllPlans = async (_, res) => {
+  try {
+    const data = await db.Plan.find({});
+    res.status(200).json(data);
+  } catch (error) {
+    res.status(500).json(error.message);
```

Comment:
Can you add a json like this 
```suggestion
    res.status(500).json({ error: error.message });
```